config/m4/ucx.m4

```diff
@@ -98,6 +98,18 @@ AS_IF([test "x$ucx_checked" != "xyes"],[
                 [AC_DEFINE([UCS_HAVE_MPOOL_PARAMS], [1], [params interface for ucs_mpool_init])],
                 [],
                 [#include <ucs/datastruct/mpool.h>])
+            AC_COMPILE_IFELSE(
```

Comment:
do we really need this check? Seems like you check if UCS_PROFILE_LOC_ID_DISABLED is defined, but then why not to use it directly in ucc_profile_on.h?

Reply:
change to directly in ucc_profile_on.h